letter/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Markup Letter</title>
+</head>
+<body>
+    <p>
+        <address class="sender-column">
+            <strong>Dr. Eleanor Gaye</strong><br>
+           Awesome Science faculty <br>
+           University of Awesome <br>
+           Bobtown, CA 99999, <br>
+           USA <br>
+           <strong>Tel: </strong> 123-456-7890 <br>
+           <strong>Email: </strong> no_reply@example.com
+        </address>   
+    </p>
+
+
+    <p class="sender-column">20 January 2016</p> 
+
+    <p>
+        <address>
+            <strong>Miss Eileen Dover</strong> <br>
+            4321 Cliff Top Edge <br>
+            Dover, CT9 XXX <br>
+            UK
+        </address>
+    </p>
+
+    <h1>Re: Eileen Dover university application</h1> 
+
+    <p>Dear Eileen,</p>
+
+    <p>Thank you for your recent application to join us at the University of Awesome's science faculty to study as part of your PhD next year. I will answer your questions one by one, in the following sections.</p> 
+
+    <h3>Starting dates</h3> 
+
+    <p>We are happy to accommodate you starting your study with us at any time, however it would suit us better if you could start at the beginning of a semester; the start dates for each one are as follows:</p> 
+
+    <ul>
+        <li>First semester: 9 September 2016</li>
+        <li>Second semester: 15 January 2017</li>
+        <li>Third semester: 2 May 2017</li>
+    </ul>
+
+    <p>Please let me know if this is ok, and if so which start date you would prefer.</p>
+
+    <p>You can find more information about <a href="">important university dates</a> on our website.</p>
```

Comment:
The use of the link tag for a link is 🙌🏾 Make sure to not have it blank though - fill the href strings with an example link i.e 
```suggestion
    <p>You can find more information about <a href="www.university.com">important university dates</a> on our website.</p>
```